app/main.ipynb

```diff
@@ -612,25 +1863,58 @@
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-03T14:14:16.230395Z",
-     "start_time": "2023-07-03T14:14:16.206501Z"
+     "end_time": "2024-11-04T21:01:47.739719Z",
+     "start_time": "2024-11-04T21:01:47.728711Z"
     }
-   }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "([-6, -3, -8, -5, -10], [1, 2, 1, 0, 8], [7, 6, 7, 12, 11])"
+      ]
+     },
+     "execution_count": 153,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 153
   },
   {
    "cell_type": "code",
-   "execution_count": 32,
-   "outputs": [],
    "source": [
-    "# write your numpy code here"
+    "def bitwise_(lst1, lst2):\n",
+    "    lst1 = np.array(lst1)\n",
+    "    lst2 = np.array(lst2)\n",
+    "    bitwise_complement = ~lst1\n",
+    "    bitwise_and = lst1 & lst2\n",
+    "    bitwise_or = lst1 | lst2\n",
+    "    return bitwise_complement, bitwise_and, bitwise_or\n",
+    "\n",
+    "result_bitwise_complement, result_bitwise_and, result_bitwise_or = unknown_signature(list1, list2)\n",
+    "result_bitwise_complement, result_bitwise_and, result_bitwise_or"
```

Comment:
The function `bitwise_` is defined but not used. The results are obtained using the `unknown_signature` function instead. Ensure that the correct function is used to obtain the results.